OutSystems/outsystems-maps
# ROU-4222: DrawCircle AddOn does not pass coordinates when editing a shape

This PR is for fix the updated coordinates not being passed to the OnDrawChange event.

### What was happening
* It didn't exist a method o get the updated shape coordinates.
* The updated coordinates were not being passed as an argument to the ShapeProviderEvent.

### What was done
* Created the getShapeProperties method for all shapes. This method returns the updated shape coordinates.
* In ShapeProviderEvent: Added an argument to trigger method to pass the updated shape coordinates.
* In AbstractDrawShape: Modified the _setOnChangeEvent method to pass the updated shape coordinates when triggering the DrawingToolsEvent.
* In AbstractProviderEvent: 
     * Created the _triggerShapeChangedEvent method that get the updated shape coordinates and trigger the Shape event. 
     * In the _buildProvider method, passed the _triggerShapeChangedEvent as the providerObjectListener handler. 

### Test Steps
1. Go to a page with Maps with DrawingTools AddIOns
2. Create a shape
3. Modify the shape
4. Check if the Coordinates and Location parameters of OnDrawingChange event changed accordingly.


### Checklist
